.github/workflows/build-and-test.yml

```diff
@@ -21,19 +21,43 @@ jobs:
     uses: StanfordSpezi/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
     with:
       runsonlabels: '["macOS", "self-hosted"]'
-      scheme: SpeziSpeech
-      artifactname: SpeziSpeech.xcresult
+      scheme: SpeziSpeech-Package
+      xcodeversion: 15.0
+      artifactname: SpeziSpeech-Package.xcresult
   ios:
     name: Build and Test iOS
     uses: StanfordSpezi/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
     with:
       runsonlabels: '["macOS", "self-hosted"]'
       path: 'Tests/UITests'
       scheme: TestApp
+      xcodeversion: 15.0
       artifactname: TestApp.xcresult
+  ipados:
+    name: Build and Test iPadOS
+    uses: StanfordSpezi/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
+    with:
+      runsonlabels: '["macOS", "self-hosted"]'
+      path: 'Tests/UITests'
+      scheme: TestApp
+      xcodeversion: 15.0
+      resultBundle: TestAppiPadOS.xcresult
+      destination: 'platform=iOS Simulator,name=iPad mini (6th generation)'
+      artifactname: TestAppiPadOS.xcresult
+  codeql:
+    name: CodeQL
+    uses: StanfordSpezi/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
+    with:
+      runsonlabels: '["macOS", "self-hosted"]'
+      codeql: true
+      scheme: SpeziSpeech-Package
+      xcodeversion: 15.0
+    permissions:
+      security-events: write
+      actions: read
```

Comment:
We currently don't allow codeql to install stuff on our own runners and therefore this is skipped there. CodeQL also currently doesn't support Swift 5.9 and therefore this action fails on the GitHub runners. We can remove it for now.
```suggestion
```